app/controllers/v1/higher_level_reviews/contestable_issues_controller.rb

```diff
@@ -15,6 +16,31 @@ def index
         )
         raise
       end
+
+      private
+
+      def merge_legacy_appeals(contestable_issues)
+        # Fetch Legacy Appels and combine with CIs
+        ci_la = nil
+        begin
+          la = decision_review_service
+               .get_legacy_appeals(user: current_user)
+               .body
+          # punch in an empty LA section if no LAs for user to distinguish no LAs from a LA call fail
+          la['data'] = [{ 'type': 'legacyAppeal', 'attributes': { 'issues': [] } }] if la['data'].empty?
+          ci_la = { "data": contestable_issues['data'] + la['data'] }
+        rescue => e
+          # If LA fails keep going Legacy Appeals are not critical, return original contestable_issues
+          log_exception_to_personal_information_log(
+            e,
+            error_class: "#{self.class.name}#index exception #{e.class} (HLR_V1_LEGACY_APPEALS)",
+            benefit_type: params[:benefit_type]
+          )
+          contestable_issues
+        else
```

Comment:
Eat-log any exceptions.  The idea here is that since this is a new endpoint, I don't want to prevent users from creating HLRs if the endpoint OR the consuming code has an issue.  

Currently, the Legacy Appeals that the new endpoint provide are not critical to creating HLRs.  Users with no Legacy Appeals can skip the Legacy Appeal Opt in question on the HLR form wizard in the website, so Legacy Appeals are not absolutely required to create an HLR.  The FE team is on board with this and can deal with a missing "legacyAppeals" section in the rendered JSON